pkg/hooks/webhook_test.go

```diff
@@ -1,17 +1,75 @@
 package hooks
 
 import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
 	"metacontroller/pkg/controller/common"
+	v11 "metacontroller/pkg/controller/composite/api/v1"
```

Comment:
doesn't matter much because it is a test but could use `compositev1`

Reply:
renamed